getting-data/introduction.md

```diff
@@ -1,6 +1,9 @@
+Here are some technical docs for my company. Can you write an improved version of this in markdown?
```

Comment:
this meant to be in here?

Reply:
haha i was using ChatGPT for recommendations